### DISCUSSION TITLE
Will my namecheap and digitalocean credits renew if I renew my github education?

### DISCUSSION THREAD

**Discussion by zayeed-bs** [community/community#64687] [GitHub Education]
### Hi everyone,

I was wondering if I renew github education next year for example, will the 1 year free namecheap .me domain  and $200 digitalocean offer renew with it. In other words, do I keep getting these benefits or are they a one time thing?

**Comment by lab596** [comment]
Looking on the DigitalOcean website it states: `This credit is available only for new users who are verified through the GitHub program as students. The credit is valid for one year from signup.` This basically means the credit is only a one-time offer and does not get renewed with GitHub education. As for the domain I found this on the website: `You get the first year of your standard offers (.me + free SSL Certificate) 100% free. After the first year, you will pay the standard annual registration costs for the domain and SSL. For example, the standard annual registration price is currently $4.88*`. This once again means that renewing the GitHub education will not keep the domain. 

Websites: 
[DigitalOcean](https://www.digitalocean.com/github-students)
[NameCheap](https://nc.me/landing/github)

TLDC answer: No, they are a one time thing
